lib/dialogs/about/index.tsx

```diff
@@ -1,125 +1,151 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Modal from 'react-modal';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import SimplenoteLogo from '../../icons/simplenote';
 import CrossIcon from '../../icons/cross';
 import TopRightArrowIcon from '../../icons/arrow-top-right';
 import Dialog from '../../dialog';
+import { closeDialog } from '../../state/ui/actions';
+
+import * as S from '../../state';
 
 const appVersion = config.version; // eslint-disable-line no-undef
 
-export class AboutDialog extends Component {
+type DispatchProps = {
+  closeDialog: () => any;
+};
+
+type Props = DispatchProps;
+
+export class AboutDialog extends Component<Props> {
   static propTypes = {
-    actions: PropTypes.object.isRequired,
-    dialog: PropTypes.object.isRequired,
-    requestClose: PropTypes.func.isRequired,
+    themeClass: PropTypes.string.isRequired,
   };
 
   render() {
-    const { dialog, requestClose } = this.props;
+    const { closeDialog, themeClass } = this.props;
     const thisYear = new Date().getFullYear();
 
     return (
-      <div className="about">
-        <Dialog hideTitleBar onDone={requestClose} title={dialog.title}>
-          <div className="about-top">
-            <SimplenoteLogo />
+      <Modal
+        key="dialog-renderer__content__about"
```

Comment:
shouldn't need the `key` in here since this we're not inside a list

Reply:
haha clearly I don't understand the rules for keys